clients/src/test/java/org/apache/kafka/common/acl/AclBindingTest.java

```diff
@@ -56,29 +58,29 @@
         new AccessControlEntryFilter(null, null, AclOperation.ANY, AclPermissionType.ANY));
 
     @Test
-    public void testMatching() throws Exception {
-        assertTrue(ACL1.equals(ACL1));
+    public void testMatching() {
```

Comment:
Took the opportunity to fix the compiler warnings in this test file, as the PR was small...